data/ocr/regex_stores.txt

```diff
@@ -6,6 +6,9 @@ Carrefour
 Carrefour||grand jury
 Colruyt
 Delhaize
+Edeka
+Eroski
+Hacendado
```

Comment:
Isn't Hacendado a brand from Mercadona? You should write Mercadona||Hacendado if so